packet/client.go

```diff
@@ -62,6 +73,7 @@ func NewClient(consumerToken, authToken string, baseURL *url.URL) (*Client, erro
 		if err != nil {
 			return nil, errors.Wrap(err, "connect to cacher")
 		}
+		wg = hg
```

Comment:
I don't think it makes sense to set the cacher client to both the `client.hardwareClient` and the `client.workflowClient`, since cacher wouldn't be dealing with workflows. And I'm also pretty sure it would panic if you tried to cast a cacher client into a tink `WorkflowSvcClient`.  
I think it's better to just leave it as nil and check for nil before fetching for workflows.

Reply:
done, pointed it to nil.